Author: Arthur Marble <arthur@info9.net>
Description: Eliminate return value in a function specified to return void (Closes: #739561)
--- cyrus-sasl2.git.orig/lib/common.c
+++ cyrus-sasl2.git/lib/common.c
@@ -841,7 +841,7 @@
 
   if (!free_mutex || free_mutex == 0x1)
     free_mutex = sasl_MUTEX_ALLOC();
-  if (!free_mutex) return SASL_FAIL;
+  if (!free_mutex) return;
 
   result = sasl_MUTEX_LOCK(free_mutex);
   if (result!=SASL_OK) return;
